### ISSUE TITLE
Wrong texture entry in \tconstruct\models\block\smeltery\casting\table.json

### ISSUE THREAD

**Issue by zISKo666** [SlimeKnights/TinkersConstruct#4439]
**Issue description:**
Wrong texture for table top in table.json

```
{
  "loader": "tconstruct:casting",
  "parent": "block/block",
  "textures": {
    "side":     "tconstruct:block/smeltery/casting/table_side",
    "particle": "tconstruct:block/smeltery/casting/table_side",
    "top":      "tconstruct:block/smeltery/casting/basin_top", <- should be table_top
    "bottom":   "tconstruct:block/smeltery/casting/table_bottom"
  },
```

**Versions:**
* Minecraft: 1.16.5
* Forge: 3.0.3.16
* Mantle: 1.6.92
* Tinkers Construct: all 1.16.5 versions including build 140

**Confirm below that this issue is not covered on the roadmap or "Whats New?"**
confirmed
**Can it be reproduced with *just* Tinkers Construct? If not, list the other mods *required* to reproduce the issue.**
Well, sure because it is just a wrong entry ;)

**Comment by StaticPH** [comment]
Huh, so it does. While it still looks okay visually, I can understand how that could be problematic for resource packs at least. And it does seem a bit lacking in contrast with such a small lip to it.

**Comment by KnightMiner** [comment]
The basin one honestly looks better than the "correct" texture (the black border makes no sense when all the sides are the light grey color on those pixels), so I just copied the basin one to the table address and updated the model

**Comment by KnightMiner** [comment]
Fixed in  3.0.3.152
